pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdTransactions.java

```diff
@@ -178,6 +178,17 @@ void run() throws Exception {
         }
     }
 
+    @Parameters(commandDescription = "Update the scale of transaction coordinators")
+    private class ScaleTransactionCoordinators extends CliCommand {
+        @Parameter(names = { "-r", "--replicas" }, description = "The scale of the transaction coordinators")
+        private int replicas;
```

Comment:
Which name is better, `partitions` or `replicas` ?

Reply:
Maybe we don't need to tangle here, I have followed Penghui's suggestion to change from `partitions` to `replicas`.
Both names are fine. How to keep it like this?